crates/services/txpool/src/txpool.rs

```diff
@@ -392,7 +384,7 @@ where
             // limit is hit, check if we can push out lowest priced tx
             let lowest_ratio = self.by_ratio_gas_tip.lowest_value().unwrap_or_default();
             if lowest_ratio >= Ratio::new(tx.tip(), tx.max_gas()) {
-                return Err(Error::NotInsertedLimitHit)
```

Comment:
Is that possible to revert all changes that added `;` please?=D

Reply:
Done.